frontend/plan/src/components/schedule/Stats.js

```diff
@@ -143,49 +137,45 @@ class Stats extends Component {
                         {" "}
                     </div>
                 </div>
-                { false
```

Comment:
Could we generalize these into two components for the gauge and the time stats? Seems like a lot of copy/paste here too.